contents.json

```diff
@@ -2130,6 +2130,11 @@
       "homepage": "https://github.com/FabrizioBrancati/BFKit-Swift",
       "tags": ["linux"]
     }, {
+      "title": "KVConstraintKit",
+      "category": "layout",
+      "description": "An Impressive Swift Autolayout DSL for iOS, tvOS & OSX.",
```

Comment:
Can you remove Swift from this line, thanks

Reply:
Sure! I did